### PR TITLE
Update/empty view

### PR DESCRIPTION
**Pull request by ag12**
**_Before:_**
before

**_After:_**
after

### CODE REVIEW

**Comment by ag12** [comment]
Yes :)

**Comment by ag12** [comment]
Agile development 😆

File: Sources/Fullscreen/EmptyView/EmptyView.swift [hunk 1]
```diff
@@ -16,7 +16,8 @@ public class EmptyView: UIView {
     private let sizeOfTriangle = CGSize(width: 90, height: 90)
     private let sizeOfCircle = CGSize(width: 75, height: 75)
     private let sizeOfRoundedSquare = CGSize(width: 55, height: 55)
-    private let sizeOfSquare = CGSize(width: 100, height: 100)
+    private let sizeOfRectangle = CGSize(width: 100, height: 100)
```
Comment (ulrikus) [inline Sources/Fullscreen/EmptyView/EmptyView.swift]: It's not really a rectangle when it's squared though..

**Comment by axldyb** [comment]
Not sure I get the reasoning behind adding one more element? It looks prette clamped now, especially on the smaller screens. The distribution of the elements also feels unbalanced. Will the design really change here? We will still use this empty screen for the MinFINN screen even if the others will have a different design.

**Comment by 3lvis** [comment]
I had the impression that it felt a little bit empty before. I like the new colors better but they might be attracting too much attention from the main element.

**Comment by axldyb** [comment]
Also, it now starts of with some movement as the circle will start a bit inside the triangle on screens with a width of 375 and smaller. I liked how it was more mysterious earlier when the user had to discover the movable elements by touching them or tilting the phone.

**Pull request merged**